scripts/tokenomics-emulation/config.json

```diff
@@ -36,7 +36,8 @@
       {
         "name": "native-tokens-transfer",
         "amount": 1,
-        "tokens_range": [1, 2]
+        "tokens_range": [1, 2],
+        "batch_count": 2
```

Comment:
batch size configuration should be 1 per all emulations. Please, move it on 1 level higher.

Reply:
Moved to higher level and passed as parameter for the emulaions.